src/components/WelcomeScreen/Login/Login.css

```diff
@@ -0,0 +1,20 @@
+.Login__field:not(:first-child) {
+  margin-top: 0.75em !important;
```

Comment:
We should avoid !important, they're contagious, if this is because you need to run over MUI styles, I'd rather you'd override at the JS level, inside the component (only for MUI related)